examples/miscellaneous/index.rst

```diff
@@ -8,5 +8,6 @@ Miscellaneous examples
    :maxdepth: 1
    :numbered:
 
-   range-timeseries
+   range
```

Comment:
I don't think this should have been changed

```suggestion
   range-timeseries
```

Reply:
@duncanmmacleod, good catch, this typo was a holdover from a previous attempt to implement all this that I just forgot to revert back. The module name is updated in commit d8dcdd0ee5bb2cc233a40ea89357183b40f24e5f.